lib/compat/wordpress-6.6/blocks.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Temporary compatibility shims for block APIs present in Gutenberg.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Adds support for excerpt to the wp_block post type.
+ */
+
+add_post_type_support( 'wp_block', 'excerpt' );
```

Comment:
Should we rename this to `post.php` as this is the name in core that creates the post types?  Also maybe call in `init` action?